x-pack/plugins/ml/public/embeddables/anomaly_charts/use_anomaly_charts_input_resolver.ts

```diff
@@ -81,9 +81,11 @@ export function useAnomalyChartsInputResolver(
 
           anomalyExplorerService.setTimeRange(timeRangeInput);
 
-          let influencersFilterQuery: InfluencersFilterQuery;
+          let influencersFilterQuery: InfluencersFilterQuery | undefined;
```

Comment:
@qn895 the current behavior of the anomaly charts embeddable is that it doesn't inherit the selection in the 'view by' swim lane when attaching to a dashboard or a case. For example, I have a job with a `status_code` influencer, and am interested in `404` errors. The charts in the Anomaly Explorer are filtered to show `status_code:404` but when I attach to a dashboard / case, this filter is no longer applied and I see results for all status codes.

Can you remember why it was implemented like this?

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/7405507/187664031-bde258bc-01cf-46cd-8e39-f2eb995a682d.png">